README.md

```diff
@@ -1,51 +1,146 @@
-# Paradise
-[![CI](https://github.com/ParadiseSS13/Paradise/workflows/CI/badge.svg)](https://github.com/ParadiseSS13/Paradise/actions?query=workflow%3ACI)
-[![Render Nanomaps](https://github.com/ParadiseSS13/Paradise/workflows/Render%20Nanomaps/badge.svg)](https://github.com/ParadiseSS13/Paradise/actions?query=workflow%3A%22Render+Nanomaps%22)
-[![Average time to resolve an issue](http://isitmaintained.com/badge/resolution/paradisess13/paradise.svg)](http://isitmaintained.com/project/paradisess13/paradise "Average time to resolve an issue")
-[![Percentage of issues still open](http://isitmaintained.com/badge/open/paradisess13/paradise.svg)](http://isitmaintained.com/project/paradisess13/paradise "Percentage of issues still open")
+<a href="#"><img src=".github/assets/paradise.png" alt="Paradise Station"></a>
 
-[![forthebadge](http://forthebadge.com/images/badges/60-percent-of-the-time-works-every-time.svg)](http://forthebadge.com)
-[![forthebadge](http://forthebadge.com/images/badges/contains-technical-debt.svg)](http://forthebadge.com)
-[![forthebadge](http://forthebadge.com/images/badges/fuck-it-ship-it.svg)](http://forthebadge.com)
+## <p align="center">Welcome to the main repository for the Paradise Station version of [Space Station 13](https://spacestation13.com/).</p>
 
-# Useful Links
+<p align="center">
+	<a href="https://github.com/ParadiseSS13/Paradise/actions?query=workflow%3ACI"><img src="https://github.com/ParadiseSS13/Paradise/workflows/CI/badge.svg" alt="CI"></a>
+	<a href="https://github.com/ParadiseSS13/Paradise/actions?query=workflow%3A%22Render+Nanomaps%22"><img src="https://github.com/ParadiseSS13/Paradise/workflows/Render%20Nanomaps/badge.svg" alt="Render Nanomaps"></a>
+	<a href="http://isitmaintained.com/project/paradisess13/paradise"><img src="http://isitmaintained.com/badge/resolution/paradisess13/paradise.svg" alt="Average time to resolve an issue"></a>
+	<a href="http://isitmaintained.com/project/paradisess13/paradise"><img src="http://isitmaintained.com/badge/open/paradisess13/paradise.svg" alt="Percentage of issues still open"></a>
+</p>
 
-- [Discord](https://discordapp.com/invite/YJDsXFE)
-- [Documentation](https://codedocs.paradisestation.org)
-- [Website](https://www.paradisestation.org/)
 
-# Useful Documents
+<p align="center">
+	<a href="https://xkcd.com/371/"><img src=".github/assets/compiles-sometimes.svg" alt="Compiles sometimes"></a>
+	<a href="https://www.reddit.com/r/SS13/comments/5oplxp/what_is_the_main_problem_with_byond_as_an_engine/dclbu1a/"><img src=".github/assets/made-in-byond.gif" alt="Made in BYOND"></a>
+	<a href="https://xkcd.com/1811/"><img src=".github/assets/made-with-crayons.svg" alt="Made with crayons"></a>
+</p>
 
-- [Autodocumentation Guide](.github/AUTODOC_GUIDE.md)
-- [Code of Conduct](./CODE_OF_CONDUCT.md)
-- [Contribution Guide](.github/CONTRIBUTING.md)
-- [Installation Guide](.github/DOWNLOADING.md)
+
+
+
+<p align="center">
+	<a href="https://discordapp.com/invite/YJDsXFE">
```

Comment:
https://discord.gg/paradisess13

this is the perma link to discord that already exists

Reply:
I'll check with the headmins

Reply:
```suggestion
	<a href="https://discord.gg/paradisess13">
```